docs/platforms/javascript/guides/nestjs/features/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: NestJS Features
+description: "Learn how Sentry's NestJS SDK exposes features for first class integration with NestJS."
+---
+
+<Note>
+  This SDK is considered **experimental and in an alpha state**. It may experience breaking changes. Please reach out on
+  [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns. This
+  SDK currently only supports [Solid](https://www.solidjs.com/) and is not yet officially compatible with
+  [Solid Start](https://start.solidjs.com/).
```

Comment:
There's a bunch of Solid.js references in here. We should replace those with NestJS.